src/components/SnazzyButton.tsx

```diff
@@ -3,21 +3,21 @@ import { Button, ButtonProps } from '@patternfly/react-core';
 import { ColoredTd } from './ColoredTd';
 
 type SnazzyButtonProps = {
-  color: string;
+
   isSnazzy: boolean;
-  customStyle?: string;
+
 } & ButtonProps;
 
-const SnazzyButton: React.FC<SnazzyButtonProps> = ({ color, isSnazzy, customStyle, ...rest }) => {
```

Comment:
you are still importing the "color" style for the below "style" variable. 

You need to import [this](https://github.com/content-services/React-typescript-demo/blob/main/src/components/helpers.ts#L1)
snazzyButtonStyling and use it WITH react-jss's useStyles ([example](https://github.com/content-services/React-typescript-demo/blob/main/src/components/Loader.tsx#L5), [example](https://github.com/content-services/React-typescript-demo/blob/main/src/components/Loader.tsx#L23))

Reply:
This issue still needs to be addressed. 
Look at the example links above along with the [CSS PRESET](https://github.com/content-services/React-typescript-demo/blob/main/src/components/helpers.ts#L1) that was intended to be used with the task from the description.